blog/content/post/2020-02-05-meta-pcengines-xen.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'XEN on the PC Engines apu2 - why you want to use it'
+abstract: 'Xen Project creates a software system that allows the execution of 
+multiple virtual guest operating systems simultaneously on a single physical machine.
```

Comment:
@Asiderr  new lines in the abstract should start with indentation, e.g. 2 spaces or tab (required for correct hugo conversion)

Reply:
@artur-rs I've added the indentation